xbmc/platform/android/network/NetworkAndroid.cpp

```diff
@@ -349,3 +366,25 @@ void CNetworkAndroid::RetrieveInterfaces()
                 n.toString());
   }
 }
+
+void CNetworkAndroid::onAvailable(const CJNINetwork n)
+{
+  CLog::Log(LOGDEBUG, "CNetworkAndroid::onAvailable The default network is now: {}", n.toString());
```

Comment:
Maybe use `CLog::LogF` and discard `CNetworkAndroid::onAvailable` from the log line?

Reply:
On Android the `LogF()` function does not display the class name, only the function name